### PR TITLE
javascript-forms - Cynthia Huynh

### PR DESCRIPTION
**Pull request by HuynhCynthia**
https://user-images.githubusercontent.com/115051720/217634338-fa024b76-3cc8-49f9-8d7c-0e7d5dafd540.mov

### CODE REVIEW

File: javascript-forms/form-values.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function handleSubmit(event) {
+  event.preventDefault();
+  const inputName = $contactForm.name.value;
+  const inputEmail = $contactForm.email.value;
+  const inputMessage = $contactForm.message.value;
```
Comment (shawnkost) [inline javascript-forms/form-values.js:5]: Try using the `elements` property of the `$contactForm` object

**Review by shawnkost — CHANGES_REQUESTED** [review]
Great job Cynthia! Just one small change requested here. What you did is completely correct, but just for the sake of following directions, I'd like you to use the property we specified in the instructions. Check out step #11

File: javascript-forms/form-values.js (hunk 1 above)
Comment (HuynhCynthia) [reply javascript-forms/form-values.js]: got it, thank you!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**